tests/unit/src/main/scala/tests/BaseAmmoniteSuite.scala

```diff
@@ -69,21 +62,21 @@ abstract class BaseAmmoniteSuite(scalaVersion: String)
       _ <- assertDefinitionAtLocation(
         "main.sc",
         "foo.as@@Json.noSpaces",
-        ".metals/readonly/io/circe/syntax/package.scala"
+        "io/circe/syntax/package.scala"
```

Comment:
Again are all these right no longer being in `.metals/readonly`? Or are they all now relative? If it's easy to switch, having the full path here might be clearer.

Reply:
Seen your reason up above. Just ignore this.